koku/api/report/aws/aws_query_handler.py

```diff
@@ -51,10 +49,11 @@ def __init__(self, query_parameters, url_data,
             tenant    (String): the tenant to use to access CUR data
             kwargs    (Dict): A dictionary for internal query alteration based on path
         """
-        kwargs['provider'] = 'AWS'
-        kwargs['no_tag_query'] = QueryFilter(operation='tags__iexact', parameter='{}')
+        if not kwargs.get('provider'):
```

Comment:
What's this check for?

Reply:
The `OCPAWSReportQueryHandler` is a subclass of the `AWSReportQueryHandler` and when it super inits, the provider was getting reset to AWS

Reply:
ah I see, thanks for the explanation.